gcc/rust/checks/errors/privacy/rust-privacy-reporter.cc

```diff
@@ -166,7 +166,12 @@ PrivacyReporter::check_for_privacy_violation (const NodeId &use_id,
     }
 
   if (!valid)
-    rust_error_at (locus, "definition is private in this context");
+    {
+      rich_location richloc (line_table, locus);
+      richloc.add_fixit_replace ("function is private");
```

Comment:
I think the error message should be more general and say item is private from glancing at the testcase changes they are not always function item privacy voilations

Reply:
Updated the `error message` to the previous message, which was added by @CohenArthur.